chia/full_node/mempool_check_conditions.py

```diff
@@ -129,11 +129,7 @@ def mempool_assert_my_amount(condition: ConditionWithArgs, unspent: CoinRecord)
 
 
 def get_name_puzzle_conditions(generator: BlockGenerator, safe_mode: bool) -> NPCResult:
-    block_program = generator.program
-    if not generator.generator_args:
-        block_program_args = NIL
-    else:
-        block_program_args = Program.from_bytes(bytes(create_generator_args(generator.generator_refs())))
+    block_program, block_program_args = setup_generator_args(generator)
```

Comment:
Why not call this method later on in the file?

Reply:
Ah, I was trying to minimize diff here, since this wasn't relevant to what I was doing at the time.

I'll move this down to right before the first use.